### DISCUSSION TITLE
How to insert many records at once?

### DISCUSSION THREAD

**Discussion by kevinxh** [prisma/prisma#4030] [Q&A] [answered]
I'm using Prisma 2 client to seed my database for integration testing.

There are no documentation around creating many records, however there are some documentation for [updateMany](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/crud#updatemany).

Would love to get some advice, thank you!

**Comment by ryands17** [comment, marked as answer]
Hey @kevinxh 👋 
Currently, there's no API to insert many records and there's a request [here](https://github.com/prisma/prisma/issues/3619) that you can follow.

As a workaround, you can batch multilpe `prisma.create` in either `Promise.all` or a [transaction](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/transactions#overview) or you could perform multiple inserts [raw query](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/raw-database-access).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!

**Comment by KaKi87** [comment]
That exists now : https://www.prisma.io/docs/orm/prisma-client/queries/transactions#batch-operations
